symengine/lib/symengine_wrapper.pyx

```diff
@@ -1395,6 +1461,126 @@ class gamma(OneArgFunction):
         cdef Basic X = sympify(x)
         return c2py(symengine.gamma(X.thisptr))
 
+class LambertW(OneArgFunction):
+    def __new__(cls, x):
+        cdef Basic X = sympify(x)
+        return c2py(symengine.lambertw(X.thisptr))
+
+    def _sympy_(self):
+        import sympy
+        return sympy.LambertW(*self.args_as_sympy())
```

Comment:
This is because `SymPy` has `LambertW` defined and not `lambertw`. The base function in `OneArgFunction` lowercases all the class names before looking up in `SymPy`. I reverted the changes for other lowercase `OneArgFunction` classes and they all work. Should I leave this as it is?